porter/v1/cluster_control_plane.proto

```diff
@@ -232,6 +232,27 @@ message ListAppRevisionsResponse {
   repeated AppRevision app_revisions = 1;
 }
 
+message PredeployStatusRequest {
+  int64 project_id = 1;
+  // app_revision_id is the id of the PorterApp that is currently installed
+  string app_revision_id = 2;
```

Comment:
small note but we've prepended app related fields with both `porter_app_*` and just `app_*`. Thinking we should go one way or the other moving forward

Reply:
Looks like we've mostly used `app_*` though

Reply:
yeah, I think app_* is the format we should move forward with